src/core/execute.h

```diff
@@ -145,6 +145,11 @@ typedef enum ExecCleanMask {
         _EXEC_CLEAN_MASK_INVALID = -1,
 } ExecCleanMask;
 
+typedef enum ExecSyscallErrorNumber {
```

Comment:
please add a comment here, that clarifies this is used as special values where syscall filters otherwise expect `errno` numbers, and also say why that's safe

Reply:
ok, moved to seccomp-util.h.